### PR TITLE
Fixes #20897: "Disable reporting" for windows method calls does produce unexpected reports

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/20897

### CODE REVIEW

File: language/src/generator/dsc/syntax.rs [hunk 1]
```diff
@@ -312,6 +314,17 @@ impl Parameters {
         self
     }
 
+    pub fn log_level(mut self, value: bool) -> Self {
```
[the remark this answers is not in the document]
Comment (amousset) [reply language/src/generator/dsc/syntax.rs:317]: ad log_level should actually be an enum I think

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Review by amousset — APPROVED** [review]

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**